src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -143,18 +143,15 @@ function getInputChunk(editor: vscode.TextEditor, chunkSize: number) {
  * @returns specifically returning undefined if the langueage is not supported,
  * otherwise true/false depending on if the language is fully supported or not belonging to the user group
  */
-function shouldFetchCrossFileContext(languageId: string, userGroup: UserGroup): boolean | undefined {
+function shouldFetchCrossFileContext(
+    languageId: vscode.TextDocument['languageId'],
+    userGroup: UserGroup
+): boolean | undefined {
     if (!isCrossFileSupported(languageId)) {
         return undefined
     }
 
-    if (languageId === 'java') {
-        return true
-    } else if (supportedLanguageToDialects[languageId] && userGroup === UserGroup.CrossFile) {
-        return true
-    } else {
-        return false
-    }
+    return true
```

Comment:
the reason we still keep returning undefined here instead of simple ture/false is because we might still need this A/B testing setup for other languages. And I want to differentiate non-supported languages return value vs. control group return value

Reply:
nit: Might be worth to make this a comment in the code

Reply:
yea Nick, I have some comments above at L143 already